datasets/ihart.yaml

```diff
@@ -0,0 +1,22 @@
+Name: iHART Whole Genome Sequencing Data Set
+Description: iHART is the [Hartwell Foundation](http://www.thehartwellfoundation.org/)’s Autism Research and Technology Initiative. This release contains whole genome data from over 1000 families with 2 or more children with autism, of which biomaterials were provided by the Autism Genetic Resource Exchange ([AGRE](http://research.agre.org/)).
+Documentation: http://www.ihart.org/data
+Contact: ihart2-org@stanford.edu
+Tags:
+ - autism spectrum disorder
+ - genomic
+ - life sciences
+ - whole genome sequencing
+License: Data use is subject to the access and publication polices of the iHART. More information on terms of use is available at [iHART website](http://www.ihart.org/)
+Resources:
+ - Description: BAM and VCF files from The iHART whole genome sequencing study.
+   ARN: arn:aws:s3:::ihart-release
+   Region: us-east-1
+   Type: S3 Bucket      
+DataAtWork:
+  Tutorials:
+  Tools & Applications:
+  Publications:
+    - Title: Inherited and *De Novo* Genetic Risk for Autism Impacts Shared Networks
+    - URL: https://www.cell.com/cell/fulltext/S0092-8674(19)30780-9
+    - AuthorName: Elizabeth K. Ruzzo *et al.*
```

Comment:
Please remove the hyphens before 'URL' and 'AuthorName' as our QC thinks that we are looking at 3 separate entries, not 3 keys for one entry!